torch/csrc/autograd/variable.h

```diff
@@ -90,7 +90,7 @@ struct TORCH_API Variable : public at::Tensor {
   //~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
The comment above in `Interface` is now out of date.

Reply:
Fixed.